acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/Eth2RequestUtils.java

```diff
@@ -533,4 +426,36 @@ private static SafeFuture<Bytes> signingRootFromSyncCommitteeUtils(
     return SafeFuture.of(
         () -> createSigningRoot.apply(ALTAIR_SPEC.getSyncCommitteeUtilRequired(slot)));
   }
+
+  public static Eth2SigningRequestBody createBlobSidecarRequest(final boolean isBlinded) {
+    final Spec spec = TestSpecFactory.createMinimal(SpecMilestone.DENEB);
+    final tech.pegasys.teku.spec.datastructures.state.ForkInfo tekuForkInfo =
+        Eth2RequestUtils.forkInfo().asInternalForkInfo();
+    final Fork tekuFork = new Fork(tekuForkInfo.getFork());
+    final tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.ForkInfo forkInfo =
+        new tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.ForkInfo(
+            tekuFork, tekuForkInfo.getGenesisValidatorsRoot());
+    final Bytes signingRoot;
+    final tech.pegasys.web3signer.core.service.http.handlers.signing.eth2.BlindedBlobSidecar
+        blindedBlobSidecar;
+    if (isBlinded) {
+      final BlindedBlobSidecar tekuBlindedBlobSidecar =
+          new DataStructureUtil(spec).randomBlindedBlobSidecar();
+      signingRoot =
+          new SigningRootUtil(spec)
+              .signingRootForBlindedBlobSidecar(tekuBlindedBlobSidecar, tekuForkInfo);
+      blindedBlobSidecar = fromInternalBlindedBlobSidecar(tekuBlindedBlobSidecar);
+    } else {
+      final BlobSidecar tekuBlobSidecar = new DataStructureUtil(spec).randomBlobSidecar();
+      signingRoot =
+          new SigningRootUtil(spec).signingRootForBlobSidecar(tekuBlobSidecar, tekuForkInfo);
+      blindedBlobSidecar = fromInternalBlobSidecar(tekuBlobSidecar);
```

Comment:
This looks like it building a blinded blob side car in both cases. Is there a difference between the two?